http/headers/expect-ct.json

```
@@ -6,7 +6,8 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Expect-CT",
           "support": {
             "chrome": {
-              "version_added": "61"
+              "version_added": "61",
+              "notes": "In version 64 invalid Expect-CT reports would be sent (even after 10 weeks had elapsed after the build date).  See <a href='https://crbug.com/786563'>bug 786563</a>."
```

```suggestion
              "notes": "Before version 65, invalid Expect-CT reports would be sent. Newer versions do not send reports after 10 weeks from the build date.  See <a href='https://crbug.com/786563'>bug 786563</a>."
```

---

Thanks for continuously following up on this PR!  I'd say the rewording you have suggested is much better, however just a little change: rather than 65, it should say 64, since a build inside of Chrome 64 resolved the issue.  Once that's done, I'd say this is ready to merge!

---

Thanks @vinyldarkscratch. That was my misinterpretation of `through` used in the mailing list email.

How's this?

```suggestion
              "notes": "Before later builds of Chrome 64, invalid Expect-CT reports would be sent. Newer versions do not send reports after 10 weeks from the build date.  See <a href='https://crbug.com/786563'>bug 786563</a>."
```

(For what it's worth, I think the whole "later builds" thing is a little bit academic. I'm reasonably sure that in practice most people are just going to check the UA and toss anything reported from Chrome <= 64 and I'd be surprised if Scott was doing something different for Report URI)

---

Yeah, that looks great, thank you very much!  For the "later builds" part, I'm going off of @jpmedley's [comment](https://github.com/mdn/browser-compat-data/pull/4472#issuecomment-510613044) of mentioning "early builds of 64".  If that's how Google wants it to be worded, let's put it that way.  :wink:

---

Yup, fair enough! :shipit:

---

I'm less concerned with how it's worded than with conveying that not all versions of 64 have the quirk.